code/_globalvars/_regexes.dm

```diff
@@ -1,4 +1,5 @@
 GLOBAL_DATUM_INIT(filename_forbidden_chars, /regex, regex(@{""|[\\\n\t/?%*:|<>]|\.\."}, "g"))
 GLOBAL_DATUM_INIT(is_color, /regex, regex("^#\[0-9a-fA-F]{6}$"))
 GLOBAL_DATUM_INIT(regex_rgb_text, /regex, regex(@"^#?(([0-9a-fA-F]{8})|([0-9a-fA-F]{6})|([0-9a-fA-F]{3}))$"))
+GLOBAL_DATUM_INIT(is_http_protocol, /regex, regex("^https?://"))
```

Comment:
imo this (and several of these tbh) is global pollution and could be just fine as a `/proc/is_http_protocol(url)` that just has the regex as a static variable inside the proc

Reply:
i personally think it's fine, these are kept to a bare minimum on paradise compared to tg